sonic-chassisd/scripts/chassisd

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
Why was this changed? #!/usr/bin/env python3 is the proper way to execute Python.

Reply:
Not supposed to be.  Restored.